### PR TITLE
Update LAN windows guide to remove Embassy copy

### PR DESCRIPTION
**Pull request by gStart9**

### CODE REVIEW

File: site/source/user-manual/connecting/connecting-lan/lan-os/lan-windows.rst [hunk 1]
```diff
@@ -1,12 +1,12 @@
 .. _lan-windows:
 
-==============================
-Trusting Embassy CA On Windows
-==============================
+====================================
+Trusting Start9 Server CA On Windows
```
Comment (kn0wmad) [inline site/source/user-manual/connecting/connecting-lan/lan-os/lan-windows.rst:4]: Just 'Trusting CA on Windows' is probably fine

Comment (kn0wmad) [inline site/source/user-manual/connecting/connecting-lan/lan-os/lan-windows.rst:4]:
```suggestion
Trusting CA On Windows
```

File: site/source/user-manual/connecting/connecting-lan/lan-os/lan-windows.rst [hunk 2]
```diff
@@ -80,12 +80,12 @@ Unfortunately, Windows does not have mDNS alias support built-in, which is neces
 
 #. Select “OK” when the import is successful.
 
-#. Verify the Embassy Local Root CA certificate is in the “Certificates” folder.
+#. Verify the your Start9 server's Local Root CA certificate is in the “Certificates” folder:
```
Comment (kn0wmad) [inline site/source/user-manual/connecting/connecting-lan/lan-os/lan-windows.rst:83]:
```suggestion
#. Verify your server's Local Root CA certificate is in the “Certificates” folder:
```

**Review by kn0wmad — CHANGES_REQUESTED** [review]
Minor nits

**Review by gStart9 — COMMENTED** [review]
thanks

**Review by kn0wmad — APPROVED** [review]

**Pull request merged**